testing/run_tests.sh

```diff
@@ -0,0 +1,49 @@
+#!/bin/bash -e
+
+#
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+set -exo errexit
```

Comment:
`#!/bin/bash -e` already equal to `set -e`.

-e  Exit immediately if a command exits with a non-zero status.
-o option-name
-x  Print commands and their arguments as they are executed.

Reply:
Thanks. `-x` removed and change `#!/bin/bash -e` to `#!/bin/bash`